Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.3.1
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential nodejs && apt-get clean
+
+# for capybara-webkit
+RUN apt-get update -qq && apt-get install -y libqt4-webkit libqt4-dev xvfb
```

Comment:
I don't think we have capybara-webkit in this project (I can just see capybara alone)